src/index.html

```diff
@@ -11,7 +11,103 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form method="post">
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label class="box">
+          Surname:
+          <input type="text" name="attribute"
+          autocomplete="off"
+          >
+        </label>
+        <label class="box">
+          Name:
+          <input type="text" name="attribute"
+          autocomplete="off"
+          >
+        </label>
+        <label class="box">
+          How old are You?
+          <input type="number" name="attribute"
+          value="12" min="1"
+          max="100"
+          >
+        </label>
+        <label class="box">
+          Full date of birth:
+          <input type="date" name="attribute">
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input type="checkbox" name="attribute"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="box">
+          E-mail:
+          <input
+            type="email"
+            name="attribute"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password: <input type="password" name="attribute">
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <label class="box">
+          Do you love cats?
+          <input type="radio" name="attribute"> Yes
+          <input type="radio" name="attribute"> No
+        </label>
+        <label class="box">
+          What is your favorite color?
+          <input type="color" name="attribute">
+        </label>
+        <label class="box">
+          What time do you go to bed?
+          <input type="time" name="attribute">
+        </label>
+        <label class="box">
+          What are your favorite brand of cars?
+          <select name="attribute" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label>
+          How do you rate our work?
+          <input type="range" name="attribute">
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Additional info:</legend>
+        <label class="box">
+          Comments:
+          <textarea
+            autocomplete="off"
+            name="attribute"
+            minlength="10"
+            maxlength="1000"
+            cols="25"
+            rows="2">
+          </textarea>
+        </label>
```

Comment:
```suggestion
            cols="25"
            rows="2"
          ></textarea>
        </label>
```

щоб виправити поведінку, яка на відео вище, треба прибрати пробіли і переноси між відкриваючим та закриваючим тегами в textarea    ></textarea>